tests/ui/associated-types/defaults-specialization.stderr

```diff
@@ -29,7 +29,7 @@ error[E0053]: method `make` has an incompatible type for trait
   --> $DIR/defaults-specialization.rs:35:18
    |
 LL |     default type Ty = bool;
-   |     ----------------------- expected this associated type
```

Comment:
Side-note: The whole point of this E0053 is that the type `bool` is *not* expected.